pytrickle/client.py

```diff
@@ -200,23 +317,83 @@ async def _ingress_loop(self):
     async def _egress_loop(self):
         """Handle outgoing frames."""
         try:
+            # If processor is running in queue mode, start a drain task to push outputs into output_queue
+            if getattr(self.frame_processor, "queue_mode", False) and not self._processor_drain_task:
+                self._processor_drain_task = asyncio.create_task(self._drain_processor_outputs())
+
             async def output_generator():
                 """Generate output frames from the queue."""
+                def _shift_audio_output(ao):
+                    # Apply a constant offset (in seconds) to each audio frame timestamp
```

Comment:
Why do we need this?

Reply:
Removed it, was part of audio buffer testing